File: solutions/0189-rotate-array/solution.java
class Solution {
    public void rotate(int[] nums, int k) {
        List<Integer>alist= new ArrayList<>();
        for(int i=0;i<nums.length;i++)
        {
            alist.add(nums[i]);
        }
        Collections.rotate(alist,k);
        for (int i = 0; i < nums.length; i++) {
            nums[i] = alist.get(i);
        }

        System.out.println(Arrays.toString(nums)); 
    }
}

File: solutions/0189-rotate-array/solution.c
void rotate(int* nums, int numsSize, int k) {
        int m=0,n=numsSize;
        k=k%n;
        int arr[n];
        for(int i=n-k;i<n;i++)
            arr[m++]=nums[i];
        for(int i=0;i<n-k;i++)
            arr[m++]=nums[i];
        for(int i=0;i<n;i++)
            nums[i]=arr[i];
}
